src/types/ant.ts

```diff
@@ -64,10 +78,10 @@ export type SortedANTRecords = Record<string, SortedANTRecord>;
 
 export const AntRecordsSchema = z.record(z.string(), AntRecordSchema);
 export const AntControllersSchema = z.array(
-  ArweaveTxIdSchema.describe('Controller address'),
+  AOAddressSchema.describe('Controller address'),
```

Comment:
...what if we relaxed this to just be any string?

Reply:
https://github.com/ar-io/ar-io-sdk/pull/431/commits/090c799a9b366960a6f29d6d9c89d78c086ce444